packages/flutter/lib/src/material/magnifier.dart

```diff
@@ -0,0 +1,331 @@
+          ? TextMagnifier.jumpBetweenLinesAnimationDuration
+          : Duration.zero,
+      child: Magnifier(
+        additionalFocalPointOffset: _extraFocalPointOffset,
+      ),
+    );
+  }
+}
+
+/// A Material styled magnifying glass.
+///
+/// {@macro flutter.widgets.magnifier.intro}
+///
+/// This widget focuses on mimicking the _style_ of the magnifier on material. For a
+/// widget that is focused on mimicking the behavior of a material magnifier, see [TextMagnifier].
+class Magnifier extends StatelessWidget {
+  /// Creates a [RawMagnifier] in the Material style.
+  ///
+  /// {@macro widgets.material.magnifier.androidDisclaimer}
+  const Magnifier({
+    super.key,
+    this.filmColor = const Color.fromARGB(8, 158, 158, 158),
+    this.additionalFocalPointOffset = Offset.zero,
+    this.borderRadius = const BorderRadius.all(Radius.circular(_borderRadius)),
+    this.size = Magnifier.kDefaultMagnifierSize,
+    this.shadows = const <BoxShadow>[
+      BoxShadow(
+          blurRadius: 1.5,
+          offset: Offset(0, 2),
+          spreadRadius: 0.75,
+          color: Color.fromARGB(25, 0, 0, 0))
+    ],
+  });
+
+  /// The default size of this [Magnifier].
+  ///
+  /// The size of the magnifier may be modified through the constructor;
+  /// [kDefaultMagnifierSize] is extracted from the default parameter of
+  /// [Magnifier]'s constructor so that positioners may depend on it.
+  @visibleForTesting
+  static const Size kDefaultMagnifierSize = Size(77.37, 37.9);
+
+  /// The vertical distance that the magnifier should be above the focal point.
+  ///
+  /// [kStandardVerticalFocalPointShift] is an unmodifiable constant so that positioning of this
+  /// [Magnifier] can be done with a garunteed size, as opposed to an estimate.
+  @visibleForTesting
+  static const double kStandardVerticalFocalPointShift = -18;
+
+  /// The color to tint the image in this [Magnifier].
+  ///
+  /// On native Android, there is a almost transparent gray tint to the
+  /// magnifier, in orderoto better distinguish the contents of the lens from
+  /// the background.
+  final Color filmColor;
+
+  static const double _borderRadius = 40;
+  static const double _magnification = 1.25;
+
+  /// The [Size] of this [Magnifier].
+  ///
+  /// This size does not include the border.
+  final Size size;
+
+  /// The shadows for this [Magnifier].
+  final List<BoxShadow> shadows;
+
+  /// The border radius for this magnifier.
+  final BorderRadius borderRadius;
+
+  /// Any additional offset the focal point requires to "point"
+  /// to the correct place.
+  ///
+  /// This is useful for instances where the magnifier is not pointing to something
+  /// directly below it.
+  final Offset additionalFocalPointOffset;
+
+  @override
+  Widget build(BuildContext context) {
+    return RawMagnifier(
+      decoration: MagnifierDecoration(
+          shape: RoundedRectangleBorder(borderRadius: borderRadius),
+          shadows: shadows),
```

Comment:
Indent by 2 not 4, close parenthesis on the next line, and then add a trailing comma after "shadows".